pages/common/docker-image.md

```diff
@@ -0,0 +1,21 @@
+# docker image
+
+> Manage Docker images.
+> See also `docker build`, `docker import`, and `docker pull`.
+> More information: <https://docs.docker.com/engine/reference/commandline/image/>.
+
+- List local Docker images:
+
+`sudo docker image ls`
```

Comment:
`sudo` is not required IMHO.

Reply:
You're right.  It's not required.

Reply:
I was not aware of this. Updating page

Reply:
Actually @sbrl `sudo` is not required on any of the examples.  I tried them all and they work for me on macOS without `sudo`.  Have you seen [this](https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user)?

Reply:
I'll fix it anf merge now.